source/openstack/guests.rst

```diff
@@ -1,18 +1,19 @@
-.. _aarch64-openstack-guests:
+.. _openstack-guests:
 
-Building Openstack AARCH64 (arm64) Guests
-=========================================
+Building Openstack Guests
+=========================
 
-This documentation covers the steps needed to build an Openstack guests on AARCH64 (arm64) using a tool called
+This documentation covers the steps needed to build an Openstack guests using a tool called
 `packer`. In addition, we include documentation which covers how to build the images manually in case you run into
 issues with packer.
 
 Current Images
 --------------
 
-We are currently maintaining AARCH64 (arm) images.
+We are currently maintaining these images:
 
 - CentOS 7
+- CentOS 8
 - Debian 10
 - Fedora 31
 - Ubuntu 18.04
```

Comment:
I know we support other platforms on other architectures (i.e. Ubunt 16.04) so we should add those here and note which arch they are supported on currently.

Reply:
Should I make a table for this? Listing them all with pure bullet points might get confusing.

Reply:
Yeah, make a table for this please.